staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -887,6 +901,9 @@ func installAPI(s *GenericAPIServer, c *Config) {
 	if c.EnableDiscovery {
 		s.Handler.GoRestfulContainer.Add(s.DiscoveryGroupManager.WebService())
 	}
+	if c.EnableAggregatedDiscoveryEndpoint && utilfeature.DefaultFeatureGate.Enabled(features.AggregatedDiscoveryEndpoint) {
+		s.Handler.GoRestfulContainer.Add(s.DiscoveryResourceManager.WebService())
```

Comment:
Does the openapi created by this webservice get used somewhere?  If not, are we particularly happy with using gorestful?  I don't hate it, but I don't love it either.

Reply:
The openapi for this discovery endpoint does get populated in our `/openapi/v2` and `/openapi/v3` documents. Is it not recommended to add additional API endpoints with gorestful?